### PR TITLE
NXT-124 Indicator history tab

### PR DESCRIPTION
**Pull request by johndavey72**
https://nicedigital.atlassian.net/browse/NXT-124

### CODE REVIEW

File: web/src/pages/indicators/[slug]/history/index.page.tsx [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+				publishedDate: product.publishedDate,
+				lastMajorModificationDate: product.lastMajorModificationDate,
+			},
+			project: {
+				reference: project.reference,
+				title: project.title,
+				groups,
+			},
+			groupSections: groups.map(({ title }, i) => ({
+				slug: title.replace(/ /g, "-").toLowerCase() + i,
```
Comment (ediblecode) [inline web/src/pages/indicators/[slug]/history/index.page.tsx:198]: Why don't we use slugify from utils here?

File: web/src/pages/indicators/[slug]/history/index.page.test.tsx [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				(await getServerSideProps(context)) as {
+					props: HistoryPageProps;
+				}
+			).props;
+		});
+
+		it("should match snapshot for main content", () => {
+			render(<HistoryPage {...props} />);
+			expect(document.body).toMatchSnapshot();
+			// eslint-disable-next-line testing-library/no-debugging-utils
```
Comment (ediblecode) [inline web/src/pages/indicators/[slug]/history/index.page.test.tsx:64]: Nitpick: please can you remove theses 2 commented lines?

File: web/src/pages/indicators/[slug]/history/index.page.tsx [hunk 3]
```diff
@@ -0,0 +1,203 @@
...
+		hasToolsAndResources,
+		hasHistory,
+	} = result;
+
+	if (!project) return { notFound: true };
+
+	if (!hasHistory) return { notFound: true };
+
+	const groups = historyPanels.sort(byTitleAlphabetically).map((panel) => {
+		const indevResource =
```
Comment (ediblecode) [inline web/src/pages/indicators/[slug]/history/index.page.tsx:129]: Can use `arrayify` to simplify these 2 lines here

File: web/src/pages/indicators/[slug]/history/index.page.tsx (hunk 1 above)
Comment (johndavey72) [reply web/src/pages/indicators/[slug]/history/index.page.tsx:198]: Good shout

File: web/src/pages/indicators/[slug]/history/index.page.test.tsx (hunk 2 above)
Comment (johndavey72) [reply web/src/pages/indicators/[slug]/history/index.page.test.tsx:64]: Yep, should have spotted those!

**Review by ediblecode — APPROVED** [review]

**Pull request merged**